lib/Backend/GlobOpt.cpp

```diff
@@ -9611,8 +9621,10 @@ GlobOpt::OptConstFoldUnary(
             }
             else
             {
-                isInt = false;
-                fValue = -(FloatConstType)INT32_MIN;
+                // disable const fold for Math.abs(INT32_MIN) because it causes dst to be float type which
+                // could be different with previous type spec result in LoopPrePass. The latter does int
+                // type spec for Math.abs(INT32_MIN).
+                return false;
```

Comment:
Why not simply " throw Js::RejitException(RejitReason::AggressiveIntTypeSpecDisabled);" here?  We will bailout anyway...